slack-plugin/src/main/kotlin/slack/gradle/tasks/PrintFossaDependencies.kt

```diff
@@ -36,6 +42,7 @@ import org.gradle.api.tasks.OutputFile
  *
  * This task is not cacheable as it has no inputs and is run on demand.
```

Comment:
This kdoc looks out of date now.

Reply:
[280d9ef](https://github.com/slackhq/slack-gradle-plugin/pull/23/commits/280d9ef3bb000b1fb66aff79e68b5ace22737c8f)